### ISSUE TITLE
Create a medium term vision

### ISSUE THREAD

**Issue by lemireg** [FredHutch/VISCtemplates#200]
Starting point for next year's vision per @asatofh is below. Create a vision or goal for what to add / change in VISCtemplates for medium/long term.

**Part 1**
Participant cohort: 
- standard language
- standard code for sample availability table

**Part 2**
Vignettes or examples of...

Skeleton: functions

Data manipulation: 
- data manipulation
- special characters

Figures / tables:
- good layout
- graph code
- good writing
- good captions

**Comment by slager** [comment]
Is this issue mostly addressed now @lemireg via the recent hackathon discussion and the batch of more specific issues @kelliemac added after our 5 Sept VISCtemplates working group meeting?

**Comment by lemireg** [comment]
> Is this issue mostly addressed now @lemireg via the recent hackathon discussion and the batch of more specific issues @kelliemac added after our 5 Sept VISCtemplates working group meeting?

Yes! I would say so. It sounds like what we'll be doing at our next meeting once we use screenshots to brainstorm what figures and tables we want to include in vignettes (though looking at this issue I'm reminded that we didn't mention the Participant Cohort section, so maybe we can add that to our "to add to the vignettes list").

**Comment by kelliemac** [comment]
I just added a couple of comments to https://github.com/FredHutch/VISCtemplates/issues/237 in light of your suggestion, @lemireg !

**Comment by kelliemac** [comment]
@lemireg is this set of changes adequately addressed by other GitHub issues now? I think we can track the plan for what to implement when using milestones https://github.com/FredHutch/VISCtemplates/milestones

**Comment by kelliemac** [comment]
since this issue actually contains multiple tasks I think it would be preferable to create individual issues as needed and close this one
